### PR TITLE
Don't change how the test work for release yet

### PR DESCRIPTION
**Pull request by icygalz**
istio-release tests should still use run_after_success images.

### CODE REVIEW

File: prow/e2e-suite.sh [hunk 1]
```diff
@@ -73,12 +75,18 @@ for ((i=1; i<=$#; i++)); do
 done
 
 export HUB=${HUB:-"gcr.io/istio-testing"}
-export TAG="${TAG:-${GIT_SHA}}""${SINGLE_TEST}"
+export TAG="${TAG:-${GIT_SHA}}"
+
+if [[ $HUB != *"istio-release"* ]]; then
+  export TAG="${TAG:-${GIT_SHA}}""${SINGLE_TEST}"
```
Comment (utka) [inline prow/e2e-suite.sh]: Put_ between sha and test name

Comment (icygalz) [reply prow/e2e-suite.sh]: done! :)

**Pull request merged**